app/build.gradle.kts

```diff
@@ -111,7 +111,9 @@ dependencies {
     val kotlinCoroutineVersion = "1.7.3"
     val mockkVersion = "1.13.9"
     val media3Version = "1.3.0"
+    val activityVersion = "1.8.2"
 
+    implementation("androidx.activity:activity-ktx:$activityVersion")
```

Comment:
Do we really need an entirely new dependency for this 👀 ?

Reply:
Yes, otherwise we cannot use the `edgeToEdge` function. I also do do not really know which dependency is otherwise used for `ComponentActivity` but it has to be there.

Reply:
Weird, ¯\_(ツ)_/¯